sheetstorm/settings/base.py

```diff
@@ -158,7 +158,9 @@
 }
 
 
-EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+# EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+EMAIL_BACKEND = "django.core.mail.backends.filebased.EmailBackend"
+EMAIL_FILE_PATH = os.path.join(BASE_DIR, "sent_emails")
```

Comment:
Is there a requirement for keeping emails as files? I am asking because it is good to have them written to console in development, and staging/production code should use a real smtp server.

Reply:
Removed, so - done